jobs/golangapiserver/spec

```diff
@@ -0,0 +1,249 @@
+---
+name: golangapiserver
+
+templates:
+  apiserver_ctl.erb: bin/apiserver_ctl
+  apiserver.yml.erb: config/apiserver.yml
+  pre-start.erb: bin/pre-start
+
+  info.json.erb: config/info.json
+  catalog.json.erb: config/catalog.json
+
+  apiserver_ca.crt.erb: config/certs/apiserver/ca.crt
+  apiserver_server.crt.erb: config/certs/apiserver/server.crt
+  apiserver_server.key.erb: config/certs/apiserver/server.key
+
+  brokerserver_ca.crt.erb: config/certs/brokerserver/ca.crt
+  brokerserver.crt.erb: config/certs/brokerserver/server.crt
+  brokerserver.key.erb: config/certs/brokerserver/server.key
```

Comment:
I guess we made a mistake in app-autoscaler code ...
```
	r.Use(basicAuthentication.Middleware)
	r.Use(httpStatusCollectMiddleware.Collect)
	r.Get(routes.BrokerCatalogRouteName).Handler(VarsFunc(ah.GetBrokerCatalog))
	r.Get(routes.BrokerCreateInstanceRouteName).Handler(VarsFunc(ah.CreateServiceInstance))
	r.Get(routes.BrokerDeleteInstanceRouteName).Handler(VarsFunc(ah.DeleteServiceInstance))
	r.Get(routes.BrokerCreateBindingRouteName).Handler(VarsFunc(ah.BindServiceInstance))
	r.Get(routes.BrokerDeleteBindingRouteName).Handler(VarsFunc(ah.UnbindServiceInstance))
```
Given all these APIs are public API protected by basic auth,  why we need broker TLS here?